cumulusci/robotframework/tests/cumulusci/base.robot

```diff
@@ -64,6 +64,51 @@ Test Perf - Parser does not choke with no keywords - Should Fail
 Test Perf Measure Other Metric
     Set Test Metric    Max_CPU_Percent    30
 
-Test Should Generate An Error - Mismatched End Perf Timer - Should Fail
-    [Tags]      noncritical
-    End Perf Timer
+Mismatched End Perf Timer - Should Fail
```

Comment:
I think it would make sense to move all of the performance tests to a separate suite. That way you can add a tag to the suite as a whole to make it easy to run just the performance tests, or ignore the performance tests.

Reply:
We do have a suite for "performance tests" (i.e. slow tests) but these are just keywords in CumulusCI.py and Salesforce.py.

Reply:
I guess I misspoke. Wouldn't it make sense to have a separate suite for the performance test keywords? There's no reason all Salesforce.py tests have to be in a single file.

Reply:
I guess now that I look at it, base.robot isn't getting too large yet, so it's fine to leave them in cumulusci/base.robot.